content/millennium/r4/foundation/conformance.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Conformance | R4 API
+layout: api
+---
+
+# Conformance
+
+<%= render '/millennium/r4/foundation/conformance.*' %>
+
+* [Capability Statement](../foundation/conformance/capability-statement)
+* [Structure Definition](../foundation/conformance/structure-definition)
```

Comment:
I think we're missing `[Operation Definition]` here

Reply:
https://github.com/cerner/fhir.cerner.com/commit/775cdd410a50bbc809a19a30052924b93f1fd20f